dmwbs.tex

```diff
@@ -78,5 +78,6 @@ \subsection{Planning Process}\label{sect:plan}
 The DM team divides the year into two six month long \textit{cycles}, running from November through May (the ``spring cycle'') and from June through October (the ``fall cycle'').
 At the start of each cycle, the DM Leadership Team (\secref{sect:dmlt}) agrees on the detailed plan of work for the cycle, and this is loaded in to JIRA as a series of ``epics'', corresponding to projects of a few person-months duration, each with defined start and end dates and resource loading.
 The DM team records work and tracks progress against epics using JIRA; the Project Controller (\secref{role:pcon}) arranges for this information to be ingested to and made available within the PMCS.
+Finally when epics are closed the T/CAM should ensure the deliverables are mentioned/linked in the closure in Jira. The DMPM shall verify all closed epics have the defined deliverables associated with them.
```

Comment:
I think I'd drop the leading "Finally". Maybe:
> When epics are closed the T/CAM should ensure the deliverables are mentioned/linked in the associated comment on JIRA.
I'd prefer a new line for the final sentence but that's not important.

Reply:
ok - pushing update